### ISSUE TITLE
SyntaxError

### ISSUE THREAD

**Issue by elginagain** [ArturSpirin/pyPS4Controller#12]
Ran into this SyntaxError, was wondering if anyone could point me in the right direction.
![pyPS4Controller](https://user-images.githubusercontent.com/58647146/91626300-b4f14900-e962-11ea-8b70-cd892f866347.png)

**Comment by elginagain** [comment]
Looks like the problem is in controller.py line 429. I noticed that Issue [#9 listening can't be stopped](https://github.com/ArturSpirin/pyPS4Controller/issues/9) drove a revision to this file around line 429 recently, is it possible that my issue is related to Issue #9 ?

**Comment by elginagain** [comment]
Nevermind, I got it figured out. I downloaded PyCharm Python IDE and ran controller.py through the debugger first with Python 3.7 interpreter which resulted in no errors just some warnings, then I ran it through the debugger using Python 2.7 interpreter and sure enough it found a syntax error on line 429. So I took this to mean that I needed to be running Python 3.7 on my Raspberry Pi. Once I installed Python 3.7 I was able to execute test.py without issue. Everything is working well now except my button mapping for triangle, square, circle and X seem to be off by one position, but that's a different issue.
